lib/logger.js

```diff
@@ -0,0 +1,11 @@
+const opts = Object.assign({
```

Comment:
we are still creating a logger object in app.js.  Did you mean to remove that?  Because it seems redundant

Reply:
Oh yes you are right, I missed that while creating the jambonz PR. Fixed